CoreGraphFile/src/au/gov/asd/tac/constellation/graph/file/io/GraphJsonWriter.java

```diff
@@ -397,8 +395,6 @@ private void writeElements(final JsonGenerator jg, final GraphReadMethods graph,
 
                 jg.writeStartObject();
                 jg.writeNumberField(GraphFileConstants.VX_ID, vxId);
-                final ObjectMapper om = new ObjectMapper();
-                final ObjectNode root = om.createObjectNode();
```

Comment:
Wow. A core bit of code used whenever we save a file had this dead code for a long time. Thanks for this @formalhaut69.